packages/wallet/backend/src/createContainer.ts

```diff
@@ -198,7 +221,8 @@ export const createContainer = (config: Env): Container<Bindings> => {
       new PaymentPointerService({
         env: await container.resolve('env'),
         rafikiClient: await container.resolve('rafikiClient'),
-        accountService: await container.resolve('accountService')
+        accountService: await container.resolve('accountService'),
+        wmPaymentPointerService: await container.resolve('wmPaymentPointerService')
```

Comment:
Why does Rapyd need `wmPaymentPointerService`?

Reply:
it doesn't, it's a leftover. Just used it to test redis interaction